### PR TITLE
#22 fixed - Commented the unused CSS Classes

### PR DESCRIPTION
**Pull request by ankitgauri26**

### CODE REVIEW

File: assets/css/rank.css [hunk 1]
```diff
@@ -6,13 +6,13 @@
     display: none;
 }
 
-.loader {
+/* .loader {
```
Comment (MaathavanJkr) [inline assets/css/rank.css:9]: as we got a new loader this wont we needed. kindly Remove this. :)

Comment (ankitgauri26) [reply assets/css/rank.css:9]: Correct. I have commented this class. Do you want me to completely remove it?

Comment (MaathavanJkr) [reply assets/css/rank.css:9]: yes, we wont use them in the future too ig.

Comment (ankitgauri26) [reply assets/css/rank.css:9]: @MaathavanJkr - sure. I'll remove this class. Thank you for the suggestion

Comment (ankitgauri26) [reply assets/css/rank.css]: @MaathavanJkr - Removed the loader class from rank.css

**Comment by MaathavanJkr** [comment]
Good work. :) Thank you. ;)

**Comment by ankitgauri26** [comment]
@MaathavanJkr - Thank you for the support and guidance 👍

**Pull request merged**